src/bp-groups/bp-groups-template.php

```diff
@@ -420,7 +423,7 @@ function bp_has_groups( $args = '' ) {
 			'max'                => false,
 			'show_hidden'        => false,
 			'page_arg'           => 'grpage',
-			'user_id'            => bp_displayed_user_id(),
+			'user_id'            => get_current_user_id(),
```

Comment:
@shohelbuddyboss @shohel It will not work as I said in Slack. So make the changes in that shortcode as discuseed.